.github/workflows/ci.yml

```
@@ -0,0 +1,206 @@
+  push:
+    branches: [ main, develop, feature/* ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  quality-gates:
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: ./V0
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)